src/site/_data/paths/identity.json

```
@@ -48,6 +48,8 @@
           "title": "FIDO Authentication with passkeys",
           "url": "https://developers.google.com/identity/fido/"
```

Please also change these lines.
```suggestion
          "title": "Passwordless login with passkeys",
          "url": "https://developers.google.com/identity/passkeys/"
```